docs/v2/resources/ng2-translate/index.md

```diff
@@ -10,26 +10,43 @@ header_sub_title: Ionic Resources
 
 ### Installing
 
-For the latest version of Ionic 2, which runs on Angular 2 RC4, let’s install the release of NG2-Translate that’s built for Angular 2 RC4. To install the correct version, run `npm install ng2-translate@2.2.2 --save --save-exact`.
+To install NG2-Translate run `npm install ng2-translate --save`.
 
 ### Bootstrapping
 
-To use NG2-Translate, it must first be imported and added it to the `ionicBootstrap` in the application. Here’s an example of how to do this:
+To use NG2-Translate, it must first be imported and added to the imports array in the NgModule of the application. Here’s an example of how to do this:
 
 ```typescript
-import { TranslateService, TranslateLoader, TranslateStaticLoader } from 'ng2-translate/ng2-translate';
-
-ionicBootstrap(MyApp, [
-  { 
-    provide: TranslateLoader,
-    useFactory: (http: Http) => new TranslateStaticLoader(http, 'assets/i18n', '.json'),
-    deps: [Http]
-  },
-  TranslateService
-]);
+import { TranslateModule } from 'ng2-translate/ng2-translate';
+
+@NgModule({
+  imports: [
+    TranslateModule.forRoot()
+  ]
+})
+```
+
+By default this will look for your translation json files in `i18n/`, if you would like to customize this path you can create a function that returns a new
+TranslateLoader:
+
+```typescript
+export function createTranslateLoader(http: Http) {
+    return new TranslateStaticLoader(http, './assets/i18n', '.json');
+}
 ```
+and then change your `@NgModule` to look like the following example:
```

Comment:
Can we reword this. Doesnt need to be "make it look like this", instead "add the following to your NgModule's import